fleetspeak/src/client/socketservice/socketservice_windows.go

```diff
@@ -32,10 +32,12 @@ func listen(socketPath string) (net.Listener, error) {
 
 	// Ensure that the parent directory exists. wnixsocket.Listen ensures the
 	// socket file exists and is truncated.
-	if _, err := os.Lstat(parent); err != nil {
-		if !os.IsNotExist(err) {
-			return nil, fmt.Errorf("os.Lstat failed: %v", err)
-		}
+	stat, err := os.Lstat(parent)
+	if err != nil && !os.IsNotExist(err) {
+		return nil, fmt.Errorf("os.Lstat failed: %v", err)
+	}
+	if err == nil {
+		log.Info("wnix socket parent %v already exists (IsDir: %v)", parent, stat.Mode().IsDir())
```

Comment:
Compile error, I think you just need add an import line:
 log "github.com/golang/glog